10-setup-htcondor.sh

```diff
@@ -367,6 +367,13 @@ if is_true "$CONTAINER_PILOT_USE_JOB_HOOK" && [[ ! -e ${prepare_hook} ]]; then
     exit 1
 fi
 
+# use GLIDEIN_ToRetire - this is for aligning with GWMS glideins
+NOW=$(date +'%s')
+GLIDEIN_ToRetire=$(($NOW + $ACCEPT_JOBS_FOR_HOURS * 60 * 60))
+
+# Give the instance 24 hours to finish up before exiting
+GLIDEIN_ToDie=$(($GLIDEIN_ToRetire + 24 * 60 * 60))
```

Comment:
Could you make the "24 hours" a variable?  I have sites that have a hard time limit, so would like to guarantee the pilot dies within a certain amount of time.

Not sure if I prefer having to add two variables to get the total time limit, or if it would be nicer to define the ToDie time directly.

Reply:
Good point! I added a new env var named `RETIREMENT_HOURS`.